core/src/main/resources/oracle/weblogic/deploy/aliases/category_modules/SecurityConfiguration.json

```diff
@@ -1418,11 +1418,11 @@
                                                    {"version": "[10,)",       "wlst_mode": "online", "wlst_name": "RealmBootStrapVersion",               "wlst_path": "WP001", "value": {"default": "1"       }, "wlst_type": "string",         "get_method": "GET", "restart_required": "true" } ],
         "RemoteAnonymousJndiEnabled":            [ {"version": "[12.2.1.3,)", "wlst_mode": "both", "wlst_name": "RemoteAnonymous${Jndi:JNDI}Enabled",    "wlst_path": "WP001", "value": {"default": "true"    }, "wlst_type": "boolean",        "restart_required": "true"  } ],
-        "RemoteAnonymousRmiiiopEnabled":         [ {"version": "[12.2.1.4.210930,12.2.1.5]",   "wlst_mode": "online", "wlst_name": "RemoteAnonymous${Rmiiiop:RMIIIOP}Enabled", "wlst_path": "WP001", "value": {"default": "true" }, "wlst_type": "boolean"         },
-                                                   {"version": "[12.2.1.3.210929,12.2.1.4)",   "wlst_mode": "online", "wlst_name": "RemoteAnonymous${Rmiiiop:RMIIIOP}Enabled", "wlst_path": "WP001", "value": {"default": "true" }, "wlst_type": "boolean"         } ,
+        "RemoteAnonymousRmiiiopEnabled":         [ {"version": "[12.2.1.3.210630,12.2.1.4)",   "wlst_mode": "online", "wlst_name": "RemoteAnonymous${Rmiiiop:RMIIIOP}Enabled", "wlst_path": "WP001", "value": {"default": "true" }, "wlst_type": "boolean"         } ,
+                                                   {"version": "[12.2.1.4.210629,12.2.1.5]",   "wlst_mode": "online", "wlst_name": "RemoteAnonymous${Rmiiiop:RMIIIOP}Enabled", "wlst_path": "WP001", "value": {"default": "true" }, "wlst_type": "boolean"         },
                                                    {"version": "[14.1.1,)",   "wlst_mode": "both", "wlst_name": "RemoteAnonymous${Rmiiiop:RMIIIOP}Enabled", "wlst_path": "WP001", "value": {"default": "true" }, "wlst_type": "boolean"         } ],
-        "RemoteAnonymousRmit3Enabled":           [ {"version": "[12.2.1.4.210930,12.2.1.5]",   "wlst_mode": "online", "wlst_name": "RemoteAnonymous${Rmit3:RMIT3}Enabled",  "wlst_path": "WP001", "value": {"default": "true"    }, "wlst_type": "boolean"         } ,
-                                                   {"version": "[12.2.1.3.210929,12.2.1.4)",   "wlst_mode": "online", "wlst_name": "RemoteAnonymous${Rmit3:RMIT3}Enabled",  "wlst_path": "WP001", "value": {"default": "true"    }, "wlst_type": "boolean"         } ,
+        "RemoteAnonymousRmit3Enabled":           [ {"version": "[12.2.1.3.210630,12.2.1.4)",   "wlst_mode": "online", "wlst_name": "RemoteAnonymous${Rmit3:RMIT3}Enabled",  "wlst_path": "WP001", "value": {"default": "true"    }, "wlst_type": "boolean"         } ,
+                                                   {"version": "[12.2.1.4.210629,12.2.1.5]",   "wlst_mode": "online", "wlst_name": "RemoteAnonymous${Rmit3:RMIT3}Enabled",  "wlst_path": "WP001", "value": {"default": "true"    }, "wlst_type": "boolean"         } ,
```

Comment:
This version range is wrong.  It should be: "[12.2.1.4.210629,14.1.1)" to reflect the fact that it should apply to any 12.2.1.x version where x >= 5.